.yamato/test_versions.metafile

```diff
@@ -3,9 +3,10 @@
 # For each "other" test, we only run it against a single version of the
 # editor to reduce the number of yamato jobs
 test_editors:
-  - version: 2018.4
-    extra_test: llapi
   - version: 2019.4
     extra_test: gym
-  - version: 2020.2
+  - version: 2020.3
     extra_test: sensor
+# TODO change to 2021.1 soon
```

Comment:
I wanted to keep the `llapi` test running. Will branch off this and try to get 2021.1 running too.